debian/control

```diff
@@ -0,0 +1,38 @@
+Source: rocksdb
+Maintainer: G. B. Versiani <guibv@yahoo.com>
+Priority: optional
+Standards-Version: 3.9.5
+Build-Depends: debhelper (>> 9),
+ devscripts,
+ hardening-includes
+Section: libs
+Homepage: http://rocksdb.org
+
+Package: librocksdb4.4-dev
```

Comment:
I've noted some compatibility problems with the library along the time, so upgrades may affect its dependents (this is one: https://github.com/stephan-hof/pyrocksdb). So, I wanted to stay on the safe side and keep two version numbers. But you, as maintainers, will always have more control over the changes, and may opt to use a single version number when you know recent changes don't break the API.